commcare-support-library/src/main/java/org/commcare/commcaresupportlibrary/BiometricUtils.java

```diff
@@ -0,0 +1,68 @@
+package org.commcare.commcaresupportlibrary;
+
+import android.util.Base64;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+import com.google.gson.reflect.TypeToken;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BiometricUtils {
+
+    public enum BiometricIdentifier {
```

Comment:
potentially worth to put into a separate file of it's own.

Reply:
c19496bced827842183f3daeca51dc425d0ec157